content/applications/sales/sales/send_quotations/different_addresses.rst

```diff
@@ -2,74 +2,54 @@
+.. image:: different_addresses/customer-addresses-setting.png
    :align: center
    :class: img-thumbnail
-   :alt: How to enable different customer addresses on Odoo Sales?
+   :alt: Activate the Customer Addresses setting
 
-Add addresses from a quotation
-==============================
+Configure the contact form
+==========================
 
-When you create a quotation, you must add a customer. This customer can be a company or a person
-with specific billing and shipping addresses already defined and registered in the system, or it can
-be a new customer. In this case, you have to **Create and edit** the contact form for your new
-customer and link it, if necessary, to a company. In this contact form, you will be able to add,
-delete and modify invoice and delivery addresses.
+Navigate to the :guilabel:`Contacts` application, or to :menuselection:`Sales > Orders >
+Customers`, and click on a customer to open their contact form. Under the :guilabel:`Contacts &
+Addresses` tab, click :guilabel:`Add`.
 
-.. image:: different_addresses/addresses_2.png
+.. image:: different_addresses/contact-form-add-address.png
    :align: center
    :class: img-thumbnail
-   :alt: How to add addresses from a quotation on Odoo Sales?
-
-.. note::
-   If you select a customer with defined invoice and delivery addresses, Odoo will automatically use
-   them to fill in the fields. Now, if you want to change it instantly, it is possible to do so
-   directly from the quotation or the sales order.
+   :alt: Add a contact/address to the contact form
 
-Add addresses from a contact form
-=================================
+Then, select which type of address to add to the contact form (i.e. :guilabel:`Invoice Address` or
+:guilabel:`Delivery Address`).
 
-Previously, we talked about the contact form that you can fill in directly from a quotation or a
-sales order to add billing and shipping addresses to customers. But, if you go to
-:menuselection:`Sales --> Orders --> Customers`, you can create or modify every customer you want
-and add, delete or modify invoice and delivery addresses instantly there, before creating a
-quotation.
-
-.. image:: different_addresses/addresses_3.png
+.. image:: different_addresses/create-contact-window.png
    :align: center
    :class: img-thumbnail
-   :alt: How to add addresses from a contact form on Odoo Sales?
-
-Deal with different addresses
-=============================
+   :alt: Create a new contact/address on a contact form
 
-Like for the previous example, go to :menuselection:`Sales --> Orders --> Customers` and create a
-new customer. There, you can add company information but, more importantly, you can enter billing
-and shipping addresses under the **Contacts & Addresses** tab.
+Enter the address information. Then, click :guilabel:`Save & Close` to save the address and close
+the :guilabel:`Create Contact` window, or click :guilabel:`Save & New` to save this address and
+immediately input another one.
```

Comment:
Enter the address information. Then, click :guilabel:`Save & Close` to save the address, and close
the :guilabel:`Create Contact` window.

Or, click :guilabel:`Save & New` to save this address, and immediately input another one.